RecoLocalMuon/GEMSegment/plugins/GEMSegmentBuilder.cc

```diff
@@ -86,12 +86,19 @@ void GEMSegmentBuilder::build(const GEMRecHitCollection* recHits, GEMSegmentColl
 
     GEMSegmentAlgorithmBase::GEMEnsemble ensemble(std::pair<const GEMSuperChamber*,      std::map<uint32_t,const GEMEtaPartition*> >(chamber,ens));
     
+    #ifdef EDM_ML_DEBUG // have lines below only compiled when in debug mode
+    edm::LogVerbatim("GEMSegmentBuilder") << "[GEMSegmentBuilder::build] run the segment reconstruction algorithm now";
```

Comment:
changing LogVerbatim to LogTrace forbid me to compile with debug flags:
scram b -j8 USER_CXXFLAGS="-DEDM_ML_DEBUG"
resulted in several errors MessageLogger.h:

/cvmfs/cms.cern.ch/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_0_pre13/src/FWCore/MessageLogger/interface/MessageLogger.h:506:22: error: expected unqualified-id before '(' token
 #define LogTrace(id) (edm::MessageDrop::debugAlwaysSuppressed || !edm::MessageDrop::instance()->debugEnabled) ? edm::LogTrace_() : edm::LogTrace_(id)